app/components/CheckoutGuestPay.jsx

```diff
@@ -105,6 +121,8 @@ class GuestPayment extends Component {
               id="billingCity"
               name="billingCity"
               label="City"
+              error={this.state.cityErr}
+              helperText={this.state.cityHelper}
               value={this.state.billingCity}
               onChange={this.handleChange}
             />
```

Comment:
create  a generateError(this.state.inputError){
if(inputError){
  return <div>my error</div>
}
}

Reply:
disregard